src/main/java/com/powsybl/caseserver/CaseController.java

```diff
@@ -224,4 +225,13 @@ public ResponseEntity<List<CaseInfos>> getMetadata(@RequestParam("ids") List<UUI
         LOGGER.debug("get Cases metadata");
         return ResponseEntity.ok().body(caseService.getMetadata(ids));
     }
+
+    @GetMapping(value = "/cases/caseBaseName")
+    @Operation(summary = "Get case base name")
+    @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "case base name"), })
```

Comment:
use @ApiResponse directly and enhance the description
`@ApiResponse(responseCode = "200", description ="Case base name retrieved successfully")`

Reply:
Ok but i don't like the term successfully i will put only retrieved